front/src/components/Navbar/TranslationToggle.tsx

```diff
@@ -0,0 +1,31 @@
+'use client';
+import { setLanguage } from '@/app/actions';
+import { usePathname } from 'next/navigation';
+import React from 'react';
```

Comment:
Do you need this import?

React has been importing itself under the hood for a long time now, so there is no need to import “again”.